util/util.go

```diff
@@ -157,3 +123,34 @@ func ParseTaskID(s string) (string, string, error) {
 
 	return id, hub, nil
 }
+
+func GetAddrInfo() (addrs []net.IP, err error) {
```

Comment:
I think we should rename it to avoid being confused with [getaddrinfo](http://man7.org/linux/man-pages/man3/getaddrinfo.3.html).

Reply:
I named it like that intentionally (because it does the same thing). But okay.